Feature/allow user select multiple addresses

evereq opened the pull request:
-   [ ] Have you followed the [contributing guidelines](https://github.com/ever-co/ever/blob/master/.github/CONTRIBUTING.md)?
-   [ ] Have you explained what your changes do, and why they add value?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

---

Support for multiple addresses per Customer

zessu committed:
(feature/allow-user-select-multiple-addresses): Add sidenav links and select address containers

zessu committed:
(feature/allow-user-select-multiple-addresses): Create new google maps view with current user location

zessu committed:
(feature/allow-user-select-multiple-addresses): Style component
- Make view look more familiar with current design trends across android and ios

zessu committed:
(feature/allow-user-select-multiple-addresses): Allow user to Google search their location

zessu committed:
(feature/allow-user-select-multiple-addresses): Show auto complete as user searches locations

zessu committed:
(refactor/allow-user-select-multiple-addresses): Move zoom level into const

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ever-co/ever?pullRequest=639) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ever-co/ever?pullRequest=639) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ever-co/ever?pullRequest=639) it.</sub>

evereq commented:
@zessu I created a Draft pull request so we can better keep track of your changes and have a place to review them

zessu committed:
(feature/allow-user-select-multiple-addresses): Auto add user address to input fields

zessu committed:
(feature/allow-user-select-multiple-addresses): Separate new address and current address pages
- allow user to see list of all current addresses and new addresses

zessu committed:
(feature/allow-user-select-multiple-addresses): Add multiple address support to entities
- add support for multiple user addresses in IUser model & class and IuserOrder model & class

zessu committed:
refactor(ever-admin): support customer multiple addresses

zessu committed:
refactor(ever-admin): include changes to support multiple addresses

zessu committed:
refactor(ever-admin): support multiple addresses

evereq committed:
Merge pull request #780 from zessu/feature/allow-user-select-multiple-addresses

Feature/allow user select multiple addresses
